source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
`Ember.on('init', function() {` has issues:

* The event system it relies up on is not performant.
* Using events obscures the order functions are called in during hook execution.
* It is not as ES2015-safe (native classes have a super, but no events).

Fwiw, Babel make it easy to use super anyway:

```js
  init() {
    this._super(...arguments);
    // some thing
  }
```